### ISSUE TITLE
Implement TPCH query 6

### ISSUE THREAD

**Issue by andygrove** [ballista-compute/ballista#260]
Update the TPCH example to support query 6:

```sql
select
	sum(l_extendedprice * l_discount) as revenue
from
	lineitem
where
	l_shipdate >= date ':1'
	and l_shipdate < date ':1' + interval '1' year
	and l_discount between :2 - 0.01 and :2 + 0.01
	and l_quantity < :3;
```

**Comment by robotlovesyou** [comment]
I think maybe this should have been closed when #309 was merged?

**Comment by djKooks** [comment]
Seems it can be closed now 👍
